libcipher/transpo.py

```diff
@@ -0,0 +1,49 @@
+"""Transposition cipher implementation
+
+Provides functions to encrypt / decrypt a message by changing the
+character position of plain text.
+
+The Transposition decrypt function will simulate the columns and
+rows of the grid that the plain text is written on by using a list of
+strings. No of Columns is calculated by (length of the text)/key.
+key value is consider as no of rows.
```

Comment:
This is a module level docstring again. Please move implementation details to comments.